libs/SalesforceSDKCore/SalesforceSDKCore/Classes/RestAPI/SFSDKBatchResponse.h

```diff
@@ -29,6 +29,7 @@ NS_SWIFT_NAME(BatchResponse)
 @interface SFSDKBatchResponse : NSObject
 @property (nonatomic,readonly) BOOL hasErrors;
 @property (nonatomic,readonly) NSArray<id> *results;
+-(instancetype)initWith:(NSDictionary *)dict;
```

Comment:
Compositie Response had it's int in the normal header file.  I need this Batch one to be in the heard file as well so I could call it from swift.